src/main/java/gillycarpetaddons/GillyCarpetAddonsSettings.java

```diff
@@ -55,12 +55,15 @@ public class GillyCarpetAddonsSettings {
   @Rule(categories = { FEATURE, EXPERIMENTAL, GILLY })
   public static boolean spectatorPlayersUsePortals = false;
 
+  @Rule(categories = { FEATURE, SURVIVAL, GILLY })
+  public static boolean replaceableFlowers = false;
```

Comment:
As stated in the issue, this should not be a `boolean` but an enum with the values:
`false`, `all_flowers`, `one_tall_flowers`

The enum can be called `ReplaceableFlowersOptions`.

Also, please could this be put before the rule `spectatorPlayersUsePortals`. I'd like to keep alphabetical order

Reply:
sorry i just dont know the alphabet :p